client-v2/src/components/Clipboard.tsx

```diff
@@ -83,13 +83,20 @@ const StyledDragIntentContainer = styled(DragIntentContainer)`
   min-height: 100%;
 `;
 
-const SupportingDivider = styled.hr`
+const SupportingDivider = styled('hr')`
   border: 0;
   border-top: 1px solid #ccc;
   margin: 0.5em 0 0.25em;
   width: 50%;
 `;
 
+const FullDivider = styled('hr')`
+  border: 0;
+  border-top: 1px solid #ccc;
+  margin: 8px -10px 4px;
+  width: 115%;
+`;
+
```

Comment:
Design wanted a general clipboard change so there is a full-width grey line under each story in the clipboard. This `115%` stuff is hacky I know but works without changing the padding everywhere else.